### ISSUE TITLE
Response only contains the body of the lambda callback

### ISSUE THREAD

**Issue by SirIle** [activescott/serverless-http-invoker#10]
Based on the examples I expected a successful invocation response to contain fields "statusCode" and "body", but it looks like it only contains the contents of the body. On line 92 only the response from invokeWithLambdaWrapper (which uses lambda-wrapper) is returned and it looks like to be only the contents of the lambda-callback. In case of errors the response is built to contain statusCode and body. 

Should the line 92 wrap the response like "return { statusCode: 200, body: response }" or would a more complex solution with status code sniffing (that the api-gateway would do if the message contains http statuscode in the beginning in brackets) be done?

**Comment by SirIle** [comment]
On second thought this is probably as I use lambda-integration and expect the api gateway to be in the mix. I usually test through serverless-offline which emulates this functionality and so I'm not setting these values by hand. So it could be debatable if the invoker should figure out the integration type and work accordingly.

**Comment by activescott** [comment]
I agree with you that the this project is optimized for "lambda proxy integration" rather than "lambda integration". I think that is appropriate since [Proxy is the default for Serverless](https://serverless.com/framework/docs/providers/aws/events/apigateway/) as well as the fact that even [API Gateway documents "lambda integration" as "legacy technology"](https://docs.aws.amazon.com/apigateway/latest/developerguide/getting-started-with-lambda-integration.html).
